components/LocationComparisonSection/LocationComparisonSection.tsx

```diff
@@ -42,6 +43,8 @@ const LocationComparisonSection: FunctionComponent<P> = ({ themes, ...props }) =
     if (!chartIDs || chartIDs.length === 0) {
       setChartIDs(chartIDs.concat(generateUniqueRandomID([])));
     }
+    const locationString = locations.map(item => item.name).join(', ');
+    addEvent('locationsCompared', { locations: locationString });
```

Comment:
You're not reusing locationString anywhere else, so may as well inline it.

Reply:
Now inlined